app/assets/stylesheets/cieszyn.scss

```diff
@@ -9,9 +9,24 @@ body.cieszyn {
     text-align: justify;
   }
 
-  table {
+  .warranty {
+    table {
+      @extend .table;
+      @extend .table-hover;
+      @extend .table-striped;
+    }
+  }
+
+  table.prices {
     @extend .table;
-    @extend .table-hover;
-    @extend .table-striped;
+    th {
+      background-color: #c3261c;
+      color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.